test/e2e/adminapi_resources.go

```diff
@@ -83,6 +83,6 @@ var _ = Describe("[Admin API] List Azure resources action", func() {
 		}
 
 		By("comparing lists of resources")
-		Expect(actualResourceIDs).To(ConsistOf(expectedResourceIDs))
+		Expect(actualResourceIDs).To(ContainElements(expectedResourceIDs))
```

Comment:
We set up a cluster for E2Es in a specific configuration, so we know exactly what elements we are going to have in the list. 

This change basically says: "we are going to have things from `expectedResourceIDs`, but maybe more`. If we do that - this test will go out of sync very quickly: it will not be failing when we add more resources into the list. And we are already working on adding more into list resoruces: #1681 adds disk encryption set into the e2e cluster and #1775 updated the geneva action to have disk encryption set.

This change means - less confidence in the code we are testing.

> In cases where the test at test/e2e/adminapi_resources.go runs immediately after a node scale test, the adminapi_resources.go test can fail depending on the timing, because some Azure resources are still being provisioned (a NIC in this case)

We knew this might happen. We had this discussion here: https://github.com/Azure/ARO-RP/pull/1655#discussion_r693873454

We should fix the test that causes this (root cause), not the symptoms.

As a quick fix for the flake - I suggest that we disable the test for now.